cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilderTest.java

```diff
@@ -49,9 +49,7 @@
 import static java.util.Arrays.asList;
 import static java.util.Collections.emptyList;
 import static java.util.Collections.singletonList;
-import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.equalTo;
 
 @ParallelSuite
 public class KafkaBrokerConfigurationBuilderTest {
```

Comment:
I'm guessing from all the whitespace/indentation changes in this file that your IDE reformatted it. While some of those changes are OK there are places where it makes the code less readable. Typically when using next builders, for example. So it would be good to revert at least those changes.